README.md

```diff
@@ -17,6 +17,8 @@ gem 'graphlient'
 
 Create a new instance of `Graphlient::Client` with a URL and optional headers/http_options.
 
+>timeout values are set in seconds
```

Comment:
Can we document the fields below in a table instead? It's unclear what the defaults are, if any, and then whether these are seconds or ms or something else.

Reply:
Great idea, totally agree!
I have updated the readme, could you check it again please?